commands/generate-key.js

```diff
@@ -9,6 +9,10 @@ module.exports = {
         .option('yolo', {
             description: 'Do not ask for extra confirmation when using Ledger',
             type: 'boolean',
+        })
```

Comment:
This option is already available on the top level. We should not add it here again.